tests/network/vmi_networking.go

```diff
@@ -415,7 +415,10 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			deadbeafVMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), deadbeafVMI)
 			Expect(err).ToNot(HaveOccurred())
 
-			libwait.WaitUntilVMIReady(deadbeafVMI, console.LoginToAlpine)
+			const unregisterSlipImageWarning = "no Slirp network binding plugin image is set in Kubevirt config, using " +
```

Comment:
This feels very fragile to me, any change to the text will break this test.
I am not sure if it is worth using this ignore list or just ignore everything.